src/test/groovy/com/github/vkorobkov/jfixtures/loader/FixtureValueTest.groovy

```diff
@@ -61,4 +61,19 @@ class FixtureValueTest extends Specification {
         _ | 50
         _ | 100500L
     }
+
+    def "#toString returns uppercased null value"() {
+        expect:
+        new FixtureValue(null).toString() == "NULL"
+    }
+
+    def "#toString does not uppercase not null value"() {
```

Comment:
I'd remove this case since other tests (`"detects text type without text: prefix"`) checks the string type regardless of it's content